modules/videoio/src/cap_libv4l.cpp

```diff
@@ -1319,6 +1320,7 @@ static IplImage* icvRetrieveFrameCAM_V4L( CvCaptureCAM_V4L* capture, int) {
     switch(capture->imageProperties.palette) {
       case VIDEO_PALETTE_RGB24:
       case VIDEO_PALETTE_YUV420:
+      case VIDEO_PALETTE_YUYV:
```

Comment:
The v4l2 api looks better, there are only one identifier for the YUYV/YUV422, so the drivers use the same. (V4L2_PIX_FMT_YUYV) http://linuxtv.org/downloads/v4l-dvb-apis/yuv-formats.html